### ISSUE TITLE
App language should default to phone language

### ISSUE THREAD

**Issue by lognaturel** [getodk/collect#890]
#### Software and hardware versions 
Collect v1.5.1-65-g068e4ade

#### Problem description
The app language defaults to English on the first launch after #817 was introduced. It should default to the phone language.

#### Steps to reproduce the problem
1. Uninstall Collect or run a version prior to the inclusion of #817 
1. Change the phone language to a language other than English that is supported by Collect
1. Install and launch Collect

The app language will default to English.

#### Expected behavior
The app language should default to the phone language if it is supported by Collect.

#### Other information 
I believe all that needs to change is the default [here](https://github.com/abdulwd/collect/blob/deeb3da2a8a9d1cc3614c8f9f349caac4b500c82/collect_app/src/main/java/org/odk/collect/android/utilities/LocaleHelper.java#L23) so that it is based on the phone's locale rather than hardcoded to English. Additionally, there will first have to be a check to make sure that the phone language is supported by Collect. If not, English is probably the right default.

**Comment by icemc** [comment]
@lognatutel may I try it

**Comment by lognaturel** [comment]
@icemc Sure!

**Comment by icemc** [comment]
@lognaturel PR submitted

**Comment by icemc** [comment]
@lognaturel @yanokwa new PR submitted which provides the functionalities requested.
